Library/OcAppleKernelLib/PrelinkedContext.c

```diff
@@ -1328,3 +1561,36 @@ PrelinkedContextBlock (
 
   return Exclude ? PatcherExcludePrelinkedKext (Identifier, &Patcher, Context) : PatcherBlockKext (&Patcher);
 }
+
+EFI_STATUS
+PrelinkedContextBlockViaLilu (
+  IN OUT PRELINKED_CONTEXT  *Context,
+  IN     CONST CHAR8        *Identifier,
+  IN     BOOLEAN            Exclude,
+  IN     UINT8              KCKind
+  )
+{
+  LILU_BLOCK_INFO_ENTRY  *CurEntry;
+
+  ASSERT (Context != NULL);
+  ASSERT (Identifier != NULL);
+
+  if (Context->LiluBlockInfos->Header.Size + sizeof (LILU_BLOCK_INFO_ENTRY) > LILU_BLOCK_INFO_SIZE_LIMIT_VERSION_0) {
```

Comment:
While an overflow here is highly unlikely, moving `sizeof (LILU_BLOCK_INFO_ENTRY)` to the right side would mitigate the (low) risk.

Reply:
> While an overflow here is highly unlikely, moving `sizeof (LILU_BLOCK_INFO_ENTRY)` to the right side would mitigate the (low) risk.

Can you elaborate? I don't get what you mean

Reply:
``Context->LiluBlockInfos->Header.Size + sizeof (LILU_BLOCK_INFO_ENTRY) > LILU_BLOCK_INFO_SIZE_LIMIT_VERSION_0``
->
``Context->LiluBlockInfos->Header.Size > LILU_BLOCK_INFO_SIZE_LIMIT_VERSION_0 - sizeof (LILU_BLOCK_INFO_ENTRY)``

Reply:
Done